app/kudos/views.py

```diff
@@ -0,0 +1,556 @@
+# -*- coding: utf-8 -*-
+'''
+    Copyright (C) 2018 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+from __future__ import print_function, unicode_literals
+
+import json
+import logging
+import re
+
+from django.contrib import messages
+from django.contrib.postgres.search import SearchVector
+from django.contrib.staticfiles.templatetags.staticfiles import static
+from django.http import JsonResponse
+from django.shortcuts import redirect
+from django.template.response import TemplateResponse
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+from django.views.decorators.csrf import csrf_exempt
+
+from dashboard.models import Activity, Profile
+from dashboard.notifications import maybe_market_kudos_to_email
+from dashboard.utils import get_web3
+from dashboard.views import record_user_action
+from eth_utils import is_address, to_normalized_address
+from gas.utils import recommend_min_gas_price_to_confirm_in_time
+from git.utils import get_emails_master, get_github_primary_email
+from ratelimit.decorators import ratelimit
+from retail.helpers import get_ip
+from web3 import Web3
+
+from .forms import KudosSearchForm
+from .models import KudosTransfer, Token
+
+logger = logging.getLogger(__name__)
+
+confirm_time_minutes_target = 4
+
+
+def get_profile(handle):
+    try:
+        to_profile = Profile.objects.get(handle__iexact=handle)
+    except Profile.MultipleObjectsReturned:
+        to_profile = Profile.objects.filter(handle__iexact=handle).order_by('-created_on').first()
+    except Profile.DoesNotExist:
```

Comment:
`sync_profile`

Reply:
Should i use this function instead?

https://github.com/jasonrhaas/web/blob/kudos-v1/app/app/utils.py#L175

Tips appears to also be using their own `get_profile` function.  Should we change that as well?

Reply:
OK I'm leaving this as is for now.